SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -39,6 +39,7 @@ USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& O
 	, ExposedRuntimeIP(TEXT(""))
 	, bStopSpatialOnExit(false)
 	, bAutoStartLocalDeployment(true)
+	, CookAndGeneratePlatform("Win64")
```

Comment:
```suggestion
	, CookAndGeneratePlatform("WindowsServer")
```

Reply:
In my experience cooking for servers was the quickest. Also Win64 doesn't appear to be a valid target platform? https://docs.unrealengine.com/en-US/Engine/Deployment/Cooking/index.html unless this doc is old.

Reply:
This is not a valid name as far as UAT is concerned. I achieved this by specifying -TargetPlatform=Win64 -server -noclient

Reply:
Oh my bad, the UAT platform != cook commandlet platform.